Cargo.toml

```diff
@@ -2,7 +2,7 @@
 name = "socket-server-mocker"
 description = "Mock socket server in Rust, for testing various network clients."
 authors = ["Thomas Prévost"]
-version = "0.0.5"
+version = "0.1.0"
```

Comment:
Is it mature enough for `0.1.0` ? It's an open question, as you are the final user :smile:

Reply:
Most Rust projects start from 0.1.0 (cargo creates new projects this way).  The more important metric is `semver promise` - once you publish, you should use [semver checker](https://github.com/obi1kenobi/cargo-semver-checks) to ensure each of your versions indicate compatibility with the prior one